eventuate-client-java-jdbc-common-tests/src/main/java/io/eventuate/javaclient/jdbc/common/tests/CommonEventuateJdbcAccessImplTest.java

```diff
@@ -33,6 +33,7 @@
   protected void clear() {
     getEventuateJdbcStatementExecutor().update(String.format("delete from %s", getEventuateSchema().qualifyTable("events")));
     getEventuateJdbcStatementExecutor().update(String.format("delete from %s", getEventuateSchema().qualifyTable("entities")));
+    getEventuateJdbcStatementExecutor().update(String.format("delete from %s", getEventuateSchema().qualifyTable("snapshots")));
```

Comment:
This is such obvious copy/paste - please fix: `truncateTable("events")`

Reply:
Sorry, I will fix